messages/en.json

```diff
@@ -39,7 +39,7 @@
   "store/buybutton.see-cart": "View cart",
   "store/technicalspecifications.title": "Technical Specifications",
   "store/availability-subscriber.title": "This product isn't available right now",
-  "store/availability-subscriber.subscribe-label": "Let me know when this product is available again",
+  "store/availability-subscriber.subscribe-label": "I want to know when this product is available again",
```

Comment:
Maybe "...will be available again"?

Reply:
Not sure if I agree, because it sounds like the user wants to know the `date` when it will become available. 

But I agree that it can be improved, so what do you think about changing it to `I want to know when this product become available`.

Or maybe we could change the whole meaning (then I could change on the other languages as well), and change it to someting like `Subscribe to this product` or `Follow this product`. 

What do you think @kevinch ?

Reply:
I like your suggestion "I want to know when this product become available", thought of "Let me know know when this product becomes available.".